src/ui/testrtc-main.html

```diff
@@ -48,18 +48,65 @@
     <!-- Settings dialog -->
     <paper-dialog id="settings">
       <h2>Settings</h2>
-      <div class="select"><label>Audio source: <select id="audioSource"></select></label></div>
-      <div class="select"><label>Video source: <select id="videoSource"></select></label></div>
+
+      <div>
+        <h3>Media</h3>
+        <div class="select"><label>Audio source: <select id="audioSource"></select></label></div>
+        <div class="select"><label>Video source: <select id="videoSource"></select></label></div>
+      </div>
+
+      <div>
+        <h3>TURN</h3>
+        <paper-input label="URI" always-float-label placeholder="E.g.: turn:myserver.com:3478" value="{{settings.turnURI}}"></paper-input>
+        <paper-input label="Username" always-float-label value="{{settings.turnUsername}}"></paper-input>
+        <paper-input label="Credential" always-float-label value="{{settings.turnCredential}}"></paper-input>
```

Comment:
PeerConnection refers to that as a credential (not password). So I prefer to use the same terminology.